lib/logger.py

```diff
@@ -10,8 +11,7 @@ def debug(msg, if_development=True):
         info(msg)
 
 def jsonstr(obj):
-	from json import dumps
-	return dumps(obj, indent=2)
+	debug(json.dumps(obj, indent=2))
```

Comment:
is this on purpose? seems weird AF

Reply:
yes, it will log the json output only if in development mode